database/factories/GitUpdateFactory.php

```diff
@@ -6,5 +6,7 @@
 
 $factory->define(App\Models\GitUpdate::class, function (Faker $faker) {
     return [
+        'name' => $faker->name,
+        'hash' => $faker->word,
```

Comment:
This should probably be wrapped in an actual hash function, even if the input is a random word.